pricegraph/src/num.rs

```diff
@@ -44,6 +44,44 @@ pub fn u256_to_f64(u: U256) -> f64 {
     (u.low_u128() as f64) * factor
 }
 
+/// Lossy saturating conversion from a `f64` to a `U256`.
+///
+/// The conversion follows roughly the same rules as converting `f64` to other
+/// primitive integer types. Namely, the conversion of `value: f64` behaves as
+/// follows:
+/// - `NaN` => `0`
+/// - `(-∞, 0]` => `0`
+/// - `(0, u256::MAX]` => `value as u256`
+/// - `(u256::MAX, +∞)` => `u256::MAX`
+///
+/// # Note
+///
+/// Copied from <https://github.com/gnosis/ethcontract-rs/blob/d69faf37ddb7af572d916bbfc4fd7614d23d1e25/src/conv.rs#L16-L43>
+pub fn f64_to_u256(value: f64) -> U256 {
```

Comment:
Would be awesome to add this to U256 itself at some point. Should we copy the tests as well?

Reply:
Create https://github.com/gnosis/dex-services/issues/1455

Reply:
I'm on it!

https://github.com/paritytech/parity-common/pull/436